### PR TITLE
Update documentation for updated jre

### PR DESCRIPTION
**Pull request by mkangia**
## Product Description

No user facing changes

## Technical Summary

When trying to run formplayer outside docker, one gets
```
$ java -jar ./formplayer.jar
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/commcare/formplayer/Application has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
```
This is after the upgrade, https://commcare-cloud.readthedocs.io/en/latest/changelog/0069-configure-java-17-for-formplayer.html

So a more up to data version of jre needs to be installed.

### Safety story

Tested locally

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: DEV_SETUP.md [hunk 1]
```diff
@@ -126,9 +126,9 @@ NOTE: Developers on Mac OS have additional prerequisites. See the [Supplementary
 
   - **Linux**:
 
-    install `default-jre` via apt:
+    install `openjdk-17-jre` via apt:
```
Comment (shubham1g5) [inline DEV_SETUP.md:129]: Maybe a better change here is to update the heading of this section from `Java (JDK 8)` -> `Java (JDK 17)` ?

Comment (mkangia) [reply DEV_SETUP.md:129]: "may be"? that should definitely be changed, you are too polite @shubham1g5 
thanks for the catch. I will update that and I assume the change in title is along with the change I already made.

**Review by shubham1g5 — APPROVED** [review]

**Comment by mkangia** [comment]
Thanks @shubham1g5

**Comment by pxwxnvermx** [comment]
Hi @mkangia ,
The Developer Setup for mac includes installation for JDK 8 also. Can you update that too in this PR?
I can help with the changes.

**Comment by mkangia** [comment]
Hey @pxwxnvermx 
lets address that in a separate PR.

**Pull request merged**